exercises/space-age/tests/space-age.rs

```diff
@@ -0,0 +1,81 @@
+// Students,
+//
+// These tests use the `num` crate to test that floating-point numbers are within an expected
+// range.
+//
+// The first time you run `cargo test`, the Num crate will automatically be downloaded
+// and installed. More information on crates can be found at
+// https://doc.rust-lang.org/book/guessing-game.html#generating-a-secret-number
+//
+extern crate space_age;
+extern crate num;
+
+use std::fmt::Display;
+use space_age::*;
+use num::{Float, NumCast};
+
+fn assert_in_delta<T: Float + Display>(expected: T, actual: T) {
```

Comment:
Maybe there's an aspect of Rust I'm missing here. I want this function to accept both f32 and f64 and the only way I could see to do that is by using the `Float` trait in `num`

Also, the `NumCast` part of `num` comes in pretty handy.